### DISCUSSION TITLE
[@types/node-telegram-bot-api] Bot API 6.0 Missing type definitions

### DISCUSSION THREAD

**Discussion by planecore** [DefinitelyTyped/DefinitelyTyped#60116] [Issues with a @types package] [answered]
node-telegram-bot-api was updated to support Bot API 6.0 but there are still no type definitions for it.

Missing function declarations:

- `setChatMenuButton()`
- `getChatMenuButton()`
- `setMyDefaultAdministratorRights()`
- `getMyDefaultAdministratorRights()`
- `answerWebAppQuery()`

Also, some fields were renamed:

- `voice_chat_scheduled` to `video_chat_scheduled`
- `voice_chat_started` to `video_chat_started`
- `voice_chat_ended` to `video_chat_ended`
- `voice_chat_participants_invited` to `video_chat_participants_invited`

[a comment by an automated account is not shown]

**Comment by konstantin24121** [comment, marked as answer]
@planecore I already create a PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60120
